dedibox-console/account/how-to/terminate-dedibox-server.mdx

```diff
@@ -0,0 +1,31 @@
+---
```

Comment:
I think this page should be in the dedibox/dedicated-servers/ section (rather than dedibox-console/acount).

This new page also needs to be added to the menu (json file, here: https://github.com/scaleway/docs-content/blob/main/menu/navigation.json) 

If you like, I can close this PR and I'm happy to redo it myself in the right section and add it to the menu in that branch? Or feel free to do it all yourself if you prefer:-)

Reply:
ping @Jonathan-alter  :)